amazon-sqs/5.6/modules/ROOT/pages/amazon-sqs-connector-upgrade-migrate.adoc

```diff
@@ -0,0 +1,71 @@
+= Amazon SQS - Upgrade the Connector to Version 5.6 - Mule 4
+:page-aliases: connectors::amazon/amazon-sqs-connector-upgrade-migrate.adoc
+
+Upgrade Anypoint Connector for Amazon SQS to take advantage of the ability to run an app on a single node or on all nodes of a cluster.
+
+== Supported Upgrade Paths
+
+[%header,cols="50a,50a"]
+|===
+|From Version | To Version
+|5.5.0 |5.6.0
+|5.4.3 |5.6.0
+|===
+
+== Changes in This Release
+
+* Cluster node support was added back to the Receive Messages input source and works the same as it did in version 5.4.3. The input source listens for messages on all cluster nodes by default, but you can change this behavior by using the `Primary node only` field on the `Advanced` sub-tab on the `Receive messages` tab. For more information, see xref:amazon-sqs-connector-config-topics.adoc#node-behavior[Receiving Messages in Cluster Mode].
+
+* Version 5.5.0 was deprecated because cluster mode support was accidentally removed from that version.
+
+== Requirements
+
+Ensure your system meets the following requirements before you perform the upgrade:
+
+[%header%autowidth.spread]
+|===
+|Application/Service|Version
+|Mule |4.1.1 and later
+|Java|1.8.0_x
+|Anypoint Studio|7.0 and later
+|===
+
+See the xref:release-notes::connector/amazon-sqs-connector-release-notes-mule-4.adoc[Amazon SQS Connector Release Notes] for updated compatibility information for patch updates.
+
+== Upgrade Prerequisites
+
+Before you upgrade, create a backup of your files, data, and configurations in case you need to restore to them.
+
+== Upgrade Steps
+
+Follow these steps to perform the upgrade:
+
+. In Anypoint Studio, create a Mule project.
+. In the *Mule Palette* view, click *Search in Exchange*.
+. Enter `sqs` in the search field.
+. In the *Available modules* section, select *Amazon SQS Connector* and click *Add*.
+. Click *Finish*.
+. Verify that the `mule-amazon-sqs-connector` dependency version is `5.6.x` in the `pom.xml` file in the Mule project. Replace `x` with the version that corresponds to the connector you are using.
+
```

Comment:
7) If you're moving from 5.4.x or earlier version to 5.6.0, no other steps are needed.
8) If you're moving from 5.5.0 and you're using Receive Messages Source, that version will receive messages on primary node only. To obtain the same behavior, go to Receive Messages Source / Advanced Tab and check Primary Node Only 
@dhirsch1 @adam-lipka @martinapagnini @patrik-patinak